src/org/labkey/targetedms/TargetedMSSchema.java

```diff
@@ -1734,6 +1735,43 @@ public Set<String> getTableNames()
         return getAllTableNames(true);
     }
 
+    @Override
+    public QueryDefinition getQueryDef(@NotNull String queryName)
+    {
+        QueryDefinition result = super.getQueryDef(queryName);
+
+        if (result == null && StringUtils.startsWithIgnoreCase(queryName, QUERY_PTM_PERCENTS_PREFIX))
+        {
+            String runIdString = queryName.substring(QUERY_PTM_PERCENTS_PREFIX.length());
+            try
+            {
+                int runId = Integer.parseInt(runIdString);
```

Comment:
runIds are long type in targetedms

Reply:
True. It'll be a while before we hit 2 billion Skyline files imported, but best to use the data type consistently. Pushed an update.